### PR TITLE
unify expressions to make sure document consistency

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

File: zh/quick-start-create-source.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 > **注意：**
 >
-> 在创建数据源对象之前，你需要先[使用 TiUP 部署 DM 集群](deploy-a-dm-cluster-using-tiup.md)。
+> 在创建数据源之前，你需要先[使用 TiUP 部署 DM 集群](deploy-a-dm-cluster-using-tiup.md)。
 
 本文档介绍如何为 TiDB Data Migration (DM) 的数据迁移任务创建数据源。
 
-数据源对象包含了访问数据源的信息。在创建数据迁移任务之前，需要先创建任务的数据源对象。数据迁移任务需要引用对应的数据源对象来获取访问配置信息。详细的数据源管理命令请参考[管理上游数据源](manage-source.md)。
+数据源包含了访问数据源的信息。在创建数据迁移任务之前，需要先创建任务的数据源。数据迁移任务需要引用对应的数据源来获取访问配置信息。详细的数据源管理命令请参考[管理上游数据源](manage-source.md)。
```
Comment (lance6716) [inline zh/quick-start-create-source.md:14]: “数据源包含了访问数据源的信息”这一句调整避免再次提到“数据源” 🤔 

数据源包含了访问迁移任务上游的所需信息？

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: zh/quick-start-create-source.md (hunk 1 above)
Comment (Liuxiaozhen12) [inline zh/quick-start-create-source.md:14]: 这段话的先后逻辑顺序是不是有些不清晰？可否调整成：
```suggestion
数据源包含了访问迁移任务上游所需的信息。数据迁移任务需要引用对应的数据源来获取访问配置信息。因此，在创建数据迁移任务之前，需要先创建任务的数据源。详细的数据源管理命令请参考[管理上游数据源](manage-source.md)。
```

Comment (Liuxiaozhen12) [reply zh/quick-start-create-source.md:14]: 这段话的先后逻辑顺序是不是有些不清晰？可否调整成：
```suggestion
数据源包含了访问迁移任务上游所需的信息。数据迁移任务需要引用对应的数据源来获取访问配置信息。因此，在创建数据迁移任务之前，需要先创建任务的数据源。详细的数据源管理命令请参考[管理上游数据源](manage-source.md)。
```

Comment (lance6716) [reply zh/quick-start-create-source.md:14]: lgtm

**Review by lance6716 — APPROVED** [review]

**Pull request merged**